packages/babel-plugin-transform-runtime/src/definitions.js

```diff
@@ -116,14 +123,18 @@ module.exports = {
       endsWith: "string/ends-with",
       fromCodePoint: "string/from-code-point",
       includes: "string/includes",
-      padLeft: "string/pad-left",
-      padRight: "string/pad-right",
+      padLeft: "string/pad-start", // deprecated
+      padRight: "string/pad-end", // deprecated
```

Comment:
I don't think it makes sense. Here was added some new entry points and anyway old `babel-runtime` versions will not work with code transformed this `transform-runtime`.